dashboard/app/models/user_level.rb

```diff
@@ -166,7 +166,7 @@ def self.update_lockable_state(user_id, level_id, script_id, locked, readonly_an
     # no need to create a level if it's just going to be locked
     return if !user_level.persisted? && locked
 
-    # Used to set as submitted if locked and readonly, removing that 01/21
+    # Used to set as submitted if locked or readonly, removing both 01/21
```

Comment:
i don't think we need comments like this. we use our git history extensively for that purpose, so better not to muddy up the code.

Reply:
Ahh that makes sense. Done- thanks!